<dec f='include/c++/12/bits/exception_ptr.h' l='74' type='std::__exception_ptr::exception_ptr std::current_exception()'/>
<dec f='include/c++/12/bits/exception_ptr.h' l='101' type='std::__exception_ptr::exception_ptr std::current_exception()'/>
<use f='include/c++/12/bits/exception_ptr.h' l='256' u='c' c='_ZSt18make_exception_ptrT_'/>
<use f='include/c++/12/bits/nested_exception.h' l='57' u='c' c='_ZNSt16nested_exceptionC1Ev'/>
<use f='tvm/3rdparty/dmlc-core/include/dmlc/common.h' l='71' u='c' c='_ZN4dmlc12OMPException3RunET_DpT0_'/>
<use f='tvm/3rdparty/dmlc-core/include/dmlc/common.h' l='76' u='c' c='_ZN4dmlc12OMPException3RunET_DpT0_'/>
<doc f='include/c++/12/bits/exception_ptr.h' l='70'>/** Obtain an exception_ptr to the currently handled exception. If there
   *  is none, or the currently handled exception is foreign, return the null
   *  value.
   */</doc>
<use f='include/c++/12/future' l='1395' u='c' c='_ZNKSt13__future_base12_Task_setterclEv'/>
<use f='include/c++/12/future' l='1418' u='c' c='_ZNKSt13__future_base12_Task_setterIT_T0_vEclEv'/>
